src/Column/MoneyColumn.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * Symfony DataTables Bundle
+ * (c) Omines Internetbureau B.V. - https://omines.nl/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Omines\DataTablesBundle\Column;
+
+use Omines\DataTablesBundle\DataTable;
+use Omines\DataTablesBundle\Exception\MissingDependencyException;
+use Symfony\Component\Form\Extension\Core\DataTransformer\MoneyToLocalizedStringTransformer;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * MoneyColumn.
+ *
+ * @author Max Mishyn <max.mishyn@wisetiger.co.uk>
+ */
+class MoneyColumn extends AbstractColumn
+{
+    /**
+     * @var MoneyToLocalizedStringTransformer
+     */
+    private $transformer;
+
+    public function normalize($value): string
+    {
+        return $this->isRaw() ? (string) $value : $this->format(intval($value));
```

Comment:
When deriving from TextColumn this can just be `parent::normalize($this->format(intval($value))`, or the `format` function can be inlined here.